product_docs/docs/biganimal/release/using_cluster/managing_cloud_account/csp_tagging.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: "Tagging Cloud Service Providers"
+deepToC: true
+---
+
+Tags are key-value pairs you can apply to resources created on Your Cloud Account for fine-grained access control and granular cost data at the bucket level.
+
+Tags are created at the project level and managed through the Cloud Service, which is used across your cloud region.
```

Comment:
what do we mean by Cloud Service ? 
and what do we mean by `used across you cloud region` ?

Reply:
@amine-khemissi and @ivoryibuaka, I rephrased this part to make it more clear, let me know what you think. 

Cloud Service is the new name that the product team assigned to BigAnimal, and Console is the term to refer to the dashboard. I used Console here as I believe makes it more clear.